website/archiver/model.py

```diff
@@ -0,0 +1,151 @@
+import datetime
+
+from modularodm import fields
+
+from framework.mongo import ObjectId
+from framework.mongo import StoredObject
+
+from website.archiver import (
+    ARCHIVER_INITIATED,
+    ARCHIVER_SUCCESS,
+    ARCHIVER_FAILURE,
+    ARCHIVER_FAILURE_STATUSES
+)
+
+from website.addons.base import StorageAddonBase
+from website import settings
+
+class ArchiveTarget(StoredObject):
+
+    _id = fields.StringField(
+        primary=True,
+        default=lambda: str(ObjectId())
+    )
+
+    name = fields.StringField()
+
+    status = fields.StringField(default=ARCHIVER_INITIATED)
+    stat_result = fields.DictionaryField()
```

Comment:
@samchrisinger Can you please document the expected format of this field? Example: https://github.com/CenterForOpenScience/osf.io/blob/e7050dd59c0e9ea40d7a963e2195248a05022d50/framework/auth/core.py#L329-341